pytest.ini

```diff
@@ -21,9 +21,12 @@
 #    updated.
 # 4) Ignore our own PendingDeprecationWarning about update_symlinks soon to be dropped.
 #    See https://github.com/certbot/certbot/issues/6284.
+# 5) Ignore pkg_resources.declare_namespace used a large number of Google's Python
```

Comment:
Otherwise LGTM. Thanks for grabbing this Alex.
```suggestion
# 5) Ignore pkg_resources.declare_namespace used in a large number of Google's Python
```